v0/destinations/mailjet/utils.js

```diff
@@ -0,0 +1,128 @@
+const get = require("get-value");
+const ErrorBuilder = require("../../util/error");
+const {
+  constructPayload,
+  getHashFromArray,
+  getIntegrationsObj,
+  getDestinationExternalID,
+  getFieldValueFromMessage
+} = require("../../util");
+const { TRANSFORMER_METRIC } = require("../../util/constant");
+const {
+  MAPPING_CONFIG,
+  CONFIG_CATEGORIES,
+  DESTINATION,
+  ACTIONS
+} = require("./config");
+
+/**
+ * Validating payload
+ * @param {*} payload
+ * @returns
+ */
+const validatePayload = message => {
```

Comment:
This will not be required, as we are already marking it as true in mapping.json, if absent it will be handled from there only.